applications/GettingStartedWith/Toast/index.html

```diff
@@ -0,0 +1,6 @@
+<div id="container">
+    <div id="buttons">
+        <div id="show"></div>
+        <div id="custom"></div>
```

Comment:
let's make ids more specific? `show-message` and `show-custom-message` for example

Reply:
Noted